microsoft/testsuites/xfstests/xfstesting.py

```diff
@@ -522,3 +534,14 @@ def _execute_xfstests(
             timeout=self.TIME_OUT,
         )
         xfstests.check_test_results(cmd_results.stdout, log_path)
+
+    def _install_xfstests(self, node: Node) -> Xfstests:
```

Comment:
try before_case. It can raise an exception, and directly skip a case. It doesn't need to add one line in each test case.